xml/schema/Core/CustomField.xml

```diff
@@ -345,6 +345,39 @@
     <comment>Name of entity being referenced.</comment>
     <add>5.60</add>
   </field>
+  <field>
+    <name>fk_entity_on_delete</name>
+    <type>varchar</type>
+    <title>On Referenced Entity Delete</title>
+    <length>255</length>
+    <default>'set_null'</default>
+    <required>true</required>
+    <comment>Behavior if referenced entity is deleted.</comment>
+    <pseudoconstant>
+      <callback>CRM_Core_BAO_CustomField::getFkEntityOnDeleteOptions</callback>
+    </pseudoconstant>
+    <html>
+      <type>Select</type>
+    </html>
+  </field>
```

Comment:
@dontub ah, I see the problem. The api is not able to figure out if the fields are selectable during upgrade because it doesn't know since when they were added. Need to add this:
```suggestion
    <add>5.71</add>
  </field>
```